lib/services/item_manager.ts

```diff
@@ -59,15 +60,26 @@ export class ItemManager extends PureService {
   private unsubChangeObserver: any
   private observers: Observer[] = []
   private collection!: ItemCollection
+  private notesCollection!: NotesCollection;
   private systemSmartTags: SNSmartTag[]
 
   constructor(modelManager: PayloadManager) {
     super();
     this.modelManager = modelManager;
+    this.systemSmartTags = BuildSmartTags();
     this.createCollection();
     this.unsubChangeObserver = this.modelManager
-      .addObserver(ContentType.Any, this.onPayloadChange.bind(this));
-    this.systemSmartTags = BuildSmartTags();
+      .addObserver(ContentType.Any, this.setPayloads.bind(this));
+  }
+
+  private createCollection() {
+    this.collection = new ItemCollection();
+    this.collection.setDisplayOptions(ContentType.Note, CollectionSort.CreatedAt, 'dsc');
```

Comment:
Is this line still necessary?

Reply:
It is. The NotesCollection leaves sorting to the ItemCollection and only does filtering

Reply:
…Yeah the name definitely needs to change